src/appleseed/renderer/modeling/bssrdf/dipolebssrdf.h

```diff
@@ -241,7 +241,11 @@ void DipoleBSSRDF::do_prepare_inputs(
         values->m_precomputed.m_channel_pdf);
 
     // Precompute the (square of the) max radius.
-    values->m_precomputed.m_rmax2 = foundation::square(dipole_max_radius(foundation::min_value(values->m_precomputed.m_sigma_tr)));
+    values->m_precomputed.m_rmax2 =
+        foundation::square(
+            dipole_max_radius(
+                foundation::min_value(
+                    values->m_precomputed.m_sigma_tr)));
```

Comment:
Is precomputation a two step process? You seems to use one precomputated value to precompute another one. Why not precompute everything at the same time (it could have some benefit for future optimization).